### PR TITLE
Migrate Balancer's pools fees view to V2

### PR DESCRIPTION
**Pull request by mendesfabio**
author: @hedgehog-jacek 
cc: @Stefenon @markusbkoch 
@hedgehog-jacek  please review the Dune checklist:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/base_sources/arbitrum_base_sources.yml [hunk 1]
```diff
@@ -44,12 +44,11 @@ sources:
             description: "Whether the transaction was completed successfully"
           - &from
             name: from
-            description: "Wallet address that initiated the transaction"
+            description: 'Wallet address that initiated the transaction'
           - &to
             name: to
-            description: "Wallet address that received the transaction"
-          - &block_hash
```
Comment (augustog) [inline models/base_sources/arbitrum_base_sources.yml:51]: seems like dropping this line broke dbt compile.

Comment (hedgehog-jacek) [reply models/base_sources/arbitrum_base_sources.yml]: This is now fixed.

**Comment by hedgehog-jacek** [comment]
Brief comments on the purpose of your changes:

 - Migrated the Dune v1 `balancer_v2.view_pools_fees` abstraction to a v2 spellbook.
     - The original v1 abstraction is: [deprecated-dune-v1-abstractions/ethereum/balancer_v2/view_pools_fees.sql ](https://github.com/duneanalytics/spellbook/blob/34835e0deccc86df5fb0bdb538397c214e7b1fe8/deprecated-dune-v1-abstractions/ethereum/balancer_v2/view_pools_fees.sql)
     - Added a DBT uniqueness test
     - Added a DBT incremental load strategy
     - Gave attribution to `masquot`, the original author of the v1 abstraction

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributors)

Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] ~~if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')~~

**Review by hedgehog-jacek — APPROVED** [review]

**Comment by markusbkoch** [comment]
@augustog ready for review, thank you

**Review by jeff-dude — APPROVED** [review]
great submission here, team 🤝

**Pull request merged**